rdr_service/alembic/versions/15b708d94dc6_update_msg_broker_token_length.py

```diff
@@ -0,0 +1,59 @@
+"""update msg broker token length
+
+Revision ID: 15b708d94dc6
+Revises: 7dc15d7b7284, e249c8cb27a9, 5d4e56e61d6f
+Create Date: 2021-07-27 13:22:16.236487
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import rdr_service.model.utils
+
+
+from rdr_service.participant_enums import PhysicalMeasurementsStatus, QuestionnaireStatus, OrderStatus
+from rdr_service.participant_enums import WithdrawalStatus, WithdrawalReason, SuspensionStatus, QuestionnaireDefinitionStatus
+from rdr_service.participant_enums import EnrollmentStatus, Race, SampleStatus, OrganizationType, BiobankOrderStatus
+from rdr_service.participant_enums import OrderShipmentTrackingStatus, OrderShipmentStatus
+from rdr_service.participant_enums import MetricSetType, MetricsKey, GenderIdentity
+from rdr_service.model.base import add_table_history_table, drop_table_history_table
+from rdr_service.model.code import CodeType
+from rdr_service.model.site_enums import SiteStatus, EnrollingStatus, DigitalSchedulingStatus, ObsoleteStatus
+
+# revision identifiers, used by Alembic.
+revision = '15b708d94dc6'
+down_revision = ('7dc15d7b7284', 'e249c8cb27a9', '5d4e56e61d6f')
+branch_labels = None
+depends_on = None
+
+
+def upgrade(engine_name):
+    globals()["upgrade_%s" % engine_name]()
+
+
+def downgrade(engine_name):
+    globals()["downgrade_%s" % engine_name]()
+
+
+def upgrade_rdr():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.execute("ALTER TABLE Message_broker_dest_auth_info MODIFY access_token varchar(4000);")
```

Comment:
That's a large token!

Reply:
Yea, it's large, they send me a temporary token with more than 1000 chars, just incase I set this column to 4000.